WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -459,7 +459,6 @@ - (CGFloat)tableView:(UITableView *)tableView heightForFooterInSection:(NSIntege
 
 - (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath
 {
-    CGFloat width = CGRectGetWidth(self.tableView.frame);
```

Comment:
Just to confirm – this is just an unused variable?

Reply:
Yes.